diagnostic_period: 1.0
topics:
  - topic: /test/hoge
    hardware_id: hoge-hw
    keycode: HOGE001
    max_freq: 20.0
    min_freq: 10.0
    tolerance: 0.50
    window_size: 1
  - topic: /test/fuga
    hardware_id: fuga-hw
    max_freq: 30.0
    min_freq: 20.0
    tolerance: 0.50
    window_size: 1
